app/presenters/summary/html_presenter.rb

```diff
@@ -38,6 +38,15 @@ class HtmlPresenter
         supporting_evidence
         more_information
         legal_representative_details
+      ],
```

Comment:
I think we may need to introduce more structure to this configuration at some point but this is fine for now.